net/mwan3/files/usr/sbin/mwan3

```diff
@@ -61,13 +61,18 @@ ifup()
 		exit 0
 	}
 
-	json_load $(ubus -S call network.interface.$1 status)
-	json_get_vars up l3_device
+	status=$(ubus -S call network.interface.$1 status)
+	[ -n "$status" ] && {
+		json_load $status
+		json_get_vars up l3_device 
+	}
+
 	config_get enabled "$1" enabled 0
 
-	if [ "$up" -eq 1 ] \
+
+	if [ "$up" == "1" ] \
```

Comment:
Why did you change the check from '-eq' to '=='?

Reply:
It compares string type with integer type, if the length of string is zero, it causes *sh: out of rang* error on shell script.